src/index.js

```diff
@@ -51,30 +51,32 @@ const onfidoRender = (options, el, merge) => {
 }
 
 
-const stripOneCapture = ({image,documentType,id}) =>
-  documentType === undefined ? {id,image} : {id,image,documentType}
+const stripOneCapture = ({image, documentType, id, side}) => {
+  const capture = {id, image}
+  if (documentType) capture.documentType = documentType
+  if (side) capture.side = side
+  return capture
+}
 
-const stripCapturesHashToNecessaryValues = captures => mapValues(captures,
+const stripCapturesHash = captures => mapValues(captures,
   capture => capture ? stripOneCapture(capture) : null)
 
+const getCaptures = () => stripCapturesHash(events.getCaptures())
+
 function bindEvents (options) {
-  const strip = stripCapturesHashToNecessaryValues
   const eventListenersMap = {
     ready: () => { options.onReady() },
-    documentCapture: data => { options.onDocumentCapture(stripOneCapture(data)) },
-    faceCapture: data => { options.onFaceCapture(stripOneCapture(data)) },
-    complete: data => { options.onComplete(strip(data)) },
+    documentCapture: () => { options.onDocumentCapture(getCaptures().documentCapture) },
```

Comment:
You're just modifying the code here I know - but it is weird to see these as inline block statements, rather than just one-liners...